src/vs/workbench/contrib/notebook/browser/media/notebookEditorStickyScroll.css

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+.monaco-workbench .notebookOverlay .notebook-sticky-scroll-container {
+	display: flex;
+}
+
+.monaco-workbench .notebookOverlay .notebook-sticky-scroll-content {
+	display: none;
+	position: absolute;
+	top: 26px;
```

Comment:
We want to set `top` through code, based on the visibility of notebook toolbar (by reading notebookOptions I guess)